runtime/browser/web_application.cc

```diff
@@ -485,13 +484,74 @@ void WebApplication::Terminate() {
   }
 }
 
+void WebApplication::CreateWindow(common::AppControl* appcontrol) {
```

Comment:
Name is bad, 
Method name is Create window, 
But it also create web view and load url